streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/entity/User.java

```diff
@@ -70,8 +72,10 @@ public class User implements Serializable {
   @NotBlank(message = "{required}")
   private String status;
 
+  @TableField(fill = FieldFill.INSERT)
   private Date createTime;
 
+  @TableField(update = "now()", fill = FieldFill.INSERT_UPDATE)
```

Comment:
It is not advisable to use annotations. This will trigger an update whenever there is a change in the records in the table, which does not meet our expectations. The `modifyTime` is only updated when users manually modify entity info on the frontend. Often, StremaPark will modify records in the table, e.g: When the system detects a change in the  job  status. that needs to be synchronized to the database. In these cases, we do not want to update the `modifyTime`.